pokemongo_bot/stepper.py

```diff
@@ -90,6 +91,7 @@ def _walk_to(self, speed, lat, lng, alt):
             self.api.set_position(lat, lng, alt)
             self.bot.heartbeat()
             logger.log("[#] Finished walking")
+            logger.log('[#] Total distance walked: ' + str(convert(self.totalDistance, "m", "km")) + ' kilometers | ' + str(convert(self.totalDistance, "m", "mi")) + ' miles.')
```

Comment:
Please have the unit displayed be the unit set in the config
https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/config.json.example#L14